ark/utils/plot_utils.py

```diff
@@ -6,13 +6,55 @@
 import skimage.io as io
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
 from skimage.segmentation import find_boundaries
 from skimage.exposure import rescale_intensity
 
 
 # plotting functions
 
+def plot_clustering_result(labeled_image, labels_dict, title,
+                           default_val=None, cmap='tab20', figsize=(10, 10)):
```

Comment:
@ngreenwald should we be setting the default `figsize` of the image to be `(1024, 1024)`?

Reply:
No, it's the size in inches or something, not pixels. 10x10 is a good size for plotting